src/MainWindow.cpp

```diff
@@ -2393,6 +2419,20 @@ void MainWindow::showDataColumnPopupMenu(const QPoint& pos)
     popupBrowseDataHeaderMenu->exec(ui->dataTable->horizontalHeader()->mapToGlobal(pos));
 }
 
+void MainWindow::showRecordPopupMenu(const QPoint& pos)
+{
+    const QString sCurrentTable = ui->comboBrowseTable->currentText();
```

Comment:
Yes, feel free to open more pull requests whenever you find something fishy in our code :smileys: It's a very old code base, so I guess there's plenty of stuff which needs fixing. We'll definitely be thankful for any more contribution :wink: